Source/Request.swift

```diff
@@ -293,11 +293,15 @@ extension Request: CustomDebugStringConvertible {
 
             if let credentials = credentialStorage.credentials(for: protectionSpace)?.values {
                 for credential in credentials {
-                    components.append("-u \(credential.user!):\(credential.password!)")
+                    let user = credential.user ?? ""
+                    let password = credential.password ?? ""
+                    components.append("-u \(user):\(password)")
```

Comment:
Could we change both of these to `guard` or `continue`?

```swift
guard let user = credential.user, let password = credential.password else { continue }
components.append("-u \(user):\(password)")
```

Then we'll circle back on this change in a future release to add better support for other types of auth.